.github/workflows/check.yml

```
@@ -108,14 +98,16 @@ jobs:
       - name: Setup DVC
         uses: iterative/setup-dvc@v1
 
+      - name: Setup | Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: stable
+          targets: ${{ matrix.target }}
+
       - name: Install cross
         run: cargo install cross --locked
       - name: Build target
-        uses: actions-rs/cargo@v1
-        with:
-          use-cross: true
-          args: --release --target=${{ matrix.target }} --features native-tls/vendored --locked
-          command: build
+        run: cargo build --release --target=${{ matrix.target }} --features native-tls/vendored --locked
```

The intent would be to run `build` using `cross` 

```suggestion
      - name: Build target using cross
        run: cross build --release --target=${{ matrix.target }} --features native-tls/vendored --locked
```